cloudify_cli/table.py

```diff
@@ -62,6 +62,8 @@ def get_values_per_column(column, row_data):
                 row_data[column] = ','.join(row_data[column])
             elif isinstance(row_data[column], bool):
                 pass  # Taking care of False (otherwise would be changed to '')
+            elif isinstance(row_data[column], int):
+                pass  # Taking care of zero (otherwise would be changed to '')
```

Comment:
This is a behaviour change- I think the tests should detect if anything breaks though.

Reply:
Yes, sure, that's something "big".  However that's an improvement IMO, so I didn't hesitate much :)

Reply:
it's also a display-only change, and tests of course cannot say if the output looks better or worse ;)

Reply:
Agreed, I'll mark this as resolved. Just wanted to make sure we were all approaching the coming testpocalypse with open eyes!